docs/manpage.rst

```diff
@@ -267,6 +287,34 @@ An action must always be specified.
    .. versionadded:: 3.10.0
 
 
+.. option:: --describe-stored-session=UUID
+
+   Get detailed information of the session with the specified UUID.
+   The output is in JSON format.
+
+   .. versionadded:: 4.7
+
+.. option:: --describe-stored-testcases=^SESSION_UUID|TIME_PERIOD
```

Comment:
Is the `^` still valid, after the latest change?

Reply:
I just pushed the change now and updated the docs everywhere.

Reply:
Fixed it.